src/AppBundle/Controller/NamespaceController.php

```diff
@@ -1699,4 +1703,19 @@ function specialCharactersConversion($string, $forHTML=false){
 
         return $response;
     }
+
+    /**
+     * @Route("/namespace/{id}/makevisible", name="namespace_make_visible", requirements={"id"="^([0-9]+)|(namespaceID){1}$"})
+     * @param OntoNamespace $namespace
+     * @return JsonResponse
+     */
+    public function makeVisibleAction(OntoNamespace $namespace)
+    {
+        $namespace->setIsVisible(true);
```

Comment:
Il faut sécuriser l'accès à cette fonction.
$this->denyAccessUnlessGranted() est nécessaire.

Reply:
Done